src/routes/results.js

```diff
@@ -1,3 +1,79 @@
+const { mockCollection } = require('../../database/schema.js');
+
 module.exports = (req, res) => {
-  res.render('search_results');
+  let options = {};
+  const filterArray = [
+    req.query.search ? 'search-' : 'null-',
+    req.query.stage ? 'stage-' : 'null-',
+    req.query.sector ? 'sector' : 'null'
+  ].join('')
+
+  const findAll = options => {
+    mockCollection.find(options, (error, startups) => {
+      if (error) res.render('not_found')
+      else {
+        res.render('search_results', {
+          startups: startups
+        })
+      }
+    })
+  }
+
+  switch (filterArray) {
+    case 'search-stage-sector':
```

Comment:
@esraajb what do you mean to use one more variable ?? in my opinion they are pretty readable because it is telling you which filter did the user choose and the one he didn't choose is null